source/input.cpp

```diff
@@ -541,6 +542,12 @@ bool Input::Read(const std::string &fn)
         {
             read_value(ifs, nbands_sto);
         }
+        else if (strcmp("kspacing", word) == 0)
+        {
+            read_value(ifs, kspacing);
+            if (kspacing <= 0.0)
+                ModuleBase::WARNING_QUIT("Input", "kspacing must > 0");
```

Comment:
Don't use WARNING_QUIT in this function, it would cause the program stuck.

Reply:
ok